x-pack/qa/operator-privileges-tests/build.gradle

```diff
@@ -0,0 +1,38 @@
+import org.elasticsearch.gradle.info.BuildParams
+
+apply plugin: 'elasticsearch.esplugin'
+apply plugin: 'elasticsearch.java-rest-test'
+
+esplugin {
+  name 'op-test'
+  description 'An test plugin for testing hard to get internals'
+  classname 'org.elasticsearch.example.OpTestPlugin'
```

Comment:
`example` ?

Reply:
It's copied/pasted from `plugins/examples/custom-settings`. Will update accordingly.